docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md

```diff
@@ -1335,6 +1354,112 @@ Office.context.mailbox.item.displayReplyForm(
 });
 ```
 
+#### getAttachmentContentAsync(attachmentId, [options], callback) → [AttachmentContent](/javascript/api/outlook/office.attachmentcontent)
+
+Gets the specified attachment from a message or appointment and returns it as an `AttachmentContent` object.
+
+The `getAttachmentContentAsync` method gets the attachment with the specified identifier from the item. As a best practice, you should use the identifier to retrieve an attachment in the same session that the attachmentIds were retrieved with the `getAttachmentsAsync` or `item.attachments` call. In Outlook Web App and OWA for Devices, the attachment identifier is valid only within the same session. A session is over when the user closes the app, or if the user starts composing an inline form then subsequently pops out the form to continue in a separate window.
+
+##### Parameters:
+
+|Name|Type|Attributes|Description|
+|---|---|---|---|
+|`attachmentId`|String||The identifier of the attachment you want to get. The maximum length of the string is 100 characters.|
+|`options`|Object|&lt;optional&gt;|An object literal that contains one or more of the following properties.|
+|`options.asyncContext`|Object|&lt;optional&gt;|Developers can provide any object they wish to access in the callback method.|
+|`callback`|function|&lt;optional&gt;|When the method completes, the function passed in the `callback` parameter is called with a single parameter, `asyncResult`, which is an [AsyncResult](/javascript/api/office/office.asyncresult) object.|
+
+##### Requirements
+
+|Requirement|Value|
+|---|---|
+|[Minimum mailbox requirement set version](/office/dev/add-ins/reference/requirement-sets/outlook-api-requirement-sets)|Preview|
+|[Minimum permission level](https://docs.microsoft.com/outlook/add-ins/understanding-outlook-add-in-permissions)|ReadItem|
+|[Applicable Outlook mode](https://docs.microsoft.com/outlook/add-ins/#extension-points)|Compose or read|
+
+##### Returns:
+
+Type:
+[AttachmentContent](/javascript/api/outlook/office.attachmentcontent)
+
+##### Example
+
+```
```

Comment:
Add language to this code sample (```javascript```) to enable code highlighting on the page.

Reply:
Addressed in PR 532.